app/assets/javascripts/samples/sample_datatable.js.erb

```diff
@@ -163,6 +163,8 @@ function dataTableInit() {
       table.on('mousedown', function() {
         $('#samples-columns-dropdown').removeClass('open');
       });
+      // Fix compatibility of fixed table header and column names modal-tooltip
+      $('.dataTables_scrollHead').css('overflow', '');
```

Comment:
can you specify 'hidden'. It will be more clean what this piece of code do :smile: